boards/x86_64/intel64/qemu-intel64/configs/pcitest/defconfig

```diff
@@ -27,10 +27,6 @@ CONFIG_BOOT_RUNFROMEXTSRAM=y
 CONFIG_BUILTIN=y
 CONFIG_CONSOLE_SYSLOG=y
 CONFIG_DEBUG_FEATURES=y
-CONFIG_DEBUG_PCI=y
```

Comment:
It's better to leave debug options enabled for `pcitest` configuration

Reply:
done